modules/running-distributed-data-science-workloads-from-notebooks.adoc

```diff
@@ -0,0 +1,39 @@
+:_module-type: PROCEDURE
+
+[id="running-distributed-data-science-workloads-from-notebooks_{context}"]
+= Running distributed data science workloads from notebooks
+
+[role='_abstract']
+To run a distributed data science workload from a notebook, you must first clone the `codeflare-sdk` repository, and provide the link to your Ray cluster image.
+
+.Prerequisites
+* You have created a data science project that contains a workbench that is running one of the default notebook images, for example, the *Standard Data Science* notebook.
+See the table in link:{rhodsdocshome}{default-format-url}/working_on_data_science_projects/creating-and-importing-notebooks_notebooks#notebook-images-for-data-scientists[Notebook images for data scientists] for a complete list of default notebook images.
+* You have launched your notebook server and logged in to Jupyter.
+
+.Procedure
+. [One-off step] If you have not already done so, clone the `codeflare-sdk` repository as follows:
+.. In the JupyterLab interface, click *Git > Clone a Repository*.
+The "Clone a repo" dialog opens.
+.. In the *Enter the URI of the remote Git repository* field, enter `https://github.com/project-codeflare/codeflare-sdk.git` and click *Clone*.
+The `codeflare-sdk` repository is listed in the left navigation pane.
+. Run a distributed workload job as shown in the following example:
+.. In the JupyterLab interface, in the left navigation pane, double-click *codeflare-sdk*.
+.. Double-click *demo-notebooks*, and then double-click *guided-demos*.
+.. Update each example demo notebook to replace the links to the example community image with a link to your Ray cluster image.
```

Comment:
.. For each example demo notebook:
   - Replace the links to the example community image with a link to your Ray cluster image.
   - Set `instascale` to `False`.